airbyte-integrations/connectors/source-s3/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "4.8.5"
+version = "4.9.0"
```

Comment:
@clnoll - How should versioning be declared if we are using progressive-rollout for this? Should this be an "rc" version, and are there other changes needed?

Reply:
Done! ✅